docs/topics/db/models.txt

```diff
@@ -1343,11 +1343,37 @@ Field name "hiding" is not permitted
 -------------------------------------
 
 In normal Python class inheritance, it is permissible for a child class to
-override any attribute from the parent class. In Django, this is not permitted
-for attributes that are :class:`~django.db.models.Field` instances (at
-least, not at the moment). If a base class has a field called ``author``, you
-cannot create another model field called ``author`` in any class that inherits
-from that base class.
+override any attribute from the parent class. In Django, this is usually not
+permitted for attributes of :class:`~django.db.models.Model` classes that are
+:class:`~django.db.models.Field` instances. If a non-abstract model base class
+has a field called ``author``, you cannot create another model field or define
+an attribute called ``author`` in any class that inherits from that base class.
+
+.. versionchanged:: 1.9
+
+    In earlier versions, these restrictions applied
+    to abstract base classes as well.
+
+Model fields inherited from an abstract base model (where
+``Meta.abstract = False``) may be overridden with any other
+value or be removed by setting ``field_name = None``.
+
+.. warning::
+
+    Model managers are inherited from abstract base classes.
+    Overriding an inherited field which is referenced by a
+    inherited :class:`~django.db.models.Manager` may cause
+    subtle bugs. See :ref:`custom managers and model
+    inheritance <custom-managers-and-inheritance>`
+
+.. note::
+
+    Some fields define extra attributes on the model, e.g. a
+    :class:`~django.db.models.ForeignKey` defines an extra attribute with
+    ``_id`` appended to the field name, as well as ``related_name`` and
+    ``related_query_name`` on the foreign model.
+    These extra attributes cannot be overridden unless the field that defines
```

Comment:
If this sentence is meant as a separate paragraph you need a blank line above it. If not, please move the start of it to the previous line.